src/app/api/reddit/route.ts

```diff
@@ -0,0 +1,26 @@
+import { mapRedditPost } from "./service";
+
+export const GET = async function GET(req: Request) {
+    try {
+      // fetch reddit posts
+      const redditPostsJSON = await fetch(
+        `https://www.reddit.com/r/AgeofMythology/hot.json`
+      );
+      const data = await redditPostsJSON.json();
+      const redditPosts: any[] = data.data.children.map((child: any) => child.data);
+      const mappedPosts = redditPosts.filter((post) => !post.stickied).map(mapRedditPost);
+  
+      return new Response(JSON.stringify(mappedPosts), {
+        headers: {
+            "Content-Type": "application/json",
+            "Cache-Control": "public, max-age=43200, stale-while-revalidate=43200",
```

Comment:
Thanks for adding this cache! Caching this response might resolve the current issues we are seeing with the Reddit fe3ed.

Reply:
Thanks